src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/DataSetViewModels.cs

```diff
@@ -0,0 +1,87 @@
+#nullable enable
+using System;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Public.Data;
+
+/// <summary>
+/// Provides high-level information about a data set.
+/// </summary>
```

Comment:
We can keep all the docblocks in if you think they provide value, but we might just want to get rid of any that are self-explanatory in this view model and the ones below.

The public API view models need to be fully documented as these are used in Swagger doc generation. However, this is somewhat overkill for the admin, especially as we've never been in the habit of documenting our view models / requests in any great detail.

Reply:
I've left the high-level record type summaries but removed all the rest. I wouldn't mind getting rid of absolutely everything if you think it's not necessary.